src/components/Viewer/Viewer.tsx

```diff
@@ -1,78 +1,78 @@
-import React, { useEffect, useState } from "react";
-import { styled } from "@stitches/react";
-import Media from "../Media/Media";
-import Navigator from "../Navigator/Navigator";
-import Player from "components/Player/Player";
-import { label } from "../../services/iiif";
+import React, { useEffect, useState } from 'react';
+import { styled } from '@stitches/react';
+import { Manifest } from '@hyperion-framework/types';
+import Media from '../Media/Media';
+import Navigator from '../Navigator/Navigator';
+import Player from 'components/Player/Player';
+import { getLabel } from '../../services/iiif';
 
 interface Props {
-  manifest: object;
+  manifest: Manifest;
 }
 
 const Viewer: React.FC<Props> = ({ manifest }) => {
   const streamingUrl: string =
-    "https://meadow-streaming.rdc-staging.library.northwestern.edu/85/bd/1f/cd/-5/ff/6-/45/fb/-a/c5/1-/e4/56/44/6d/cb/00/6298d09f04833eb737504941812b0442e6253a4e286e79db3b11e16f9b39c604-1080.m3u8";
+    'https://meadow-streaming.rdc-staging.library.northwestern.edu/85/bd/1f/cd/-5/ff/6-/45/fb/-a/c5/1-/e4/56/44/6d/cb/00/6298d09f04833eb737504941812b0442e6253a4e286e79db3b11e16f9b39c604-1080.m3u8';
   const publicUrl: string =
-    "https://bitdash-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8";
+    'https://bitdash-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8';
 
-  return (
-    <ViewerWrapper>
-      <Header>
-        <span>
-          {label(
-            { en: ["Aliquam eu metus at arcu c.1967 sagittis mollis ut ac"] },
-            "en"
-          )}
-        </span>
-      </Header>
-      <ViewerInner>
-        <Main>
-          <Player streamingUrl={publicUrl} />
-          <Media items={{}} activeItem={0} />
-        </Main>
-        <Aside>
-          <Navigator currentTime={100} tracks={{}} />
-        </Aside>
-      </ViewerInner>
-    </ViewerWrapper>
-  );
+  if (manifest) {
```

Comment:
Totally no big deal, but an alternate way to write this not having to mess with `if else` blocks could be...more Node-ish:

```
if (!imanifest) return null;

return (
   <ViewerWrapper>
  ...
);
```

Reply:
@adamjarling Thanks for this, I've never really thought about inverting the logic there.